Doppler.HelloMicroservice/Properties/launchSettings.json

```
@@ -0,0 +1,31 @@
+﻿{
+  "$schema": "http://json.schemastore.org/launchsettings.json",
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:10693",
+      "sslPort": 0
+    }
+  },
+  "profiles": {
```

Could be nice have the `Docker` profile
```suggestion
  "profiles": {
      "Docker": {
      "commandName": "Docker",
      "launchBrowser": true,
      "launchUrl": "{Scheme}://{ServiceHost}:{ServicePort},
      "publishAllPorts": true,
      "useSSL": true,
      "environmentVariables": {
        "ASPNETCORE_ENVIRONMENT": "Development"
      }
    },
```

---

I don't usually version launchSettings.json

---

I have talked with @hrocca, we concluded that probably removing the `launchSettings.json` is useful in a running project in order to allow the developers to configure their own preferred settings. 

But by the moment keeping it could be useful as a reference.

Regarding Docker profile, I will deal with this when I test it in Visual Studio, by the moment I am using VS Code and, if I am not wrong, I cannot test it. Thanks!

---

Done! thanks!